openmc/data/multipole.py

```diff
@@ -434,7 +433,10 @@ def from_hdf5(cls, group_or_filename):
             group = group_or_filename
         else:
             h5file = h5py.File(group_or_filename, 'r')
-            version = h5file['version'].value[0].decode()
+            try:
```

Comment:
@cjosey I felt confusing here too. All the libraries I generated using openw broke when running `h5file['version'].value[0].decode()` as the `h5file['version'].value` is a string `v0.2` while it is a array `[v0.2]` for the existing libraries. I tested on h5py `2.5.0` and `2.6.0`. Do you know why? Have you changed the openw?